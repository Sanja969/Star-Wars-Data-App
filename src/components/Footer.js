import { useNavigate } from 'react-router-dom';

const Footer = () => {
  const navigate = useNavigate();
  return (
    <footer className="text-white d-flex align-items-center justify-content-around">
      <button type="button" onClick={() => navigate('/')} className="logo-footer">Star Wars</button>
      <a href="https://sanja42.pythonanywhere.com/" className="text-decoration-none text-white">COPYRIGHT Â© SANJA969</a>
    </footer>
  );
};

export default Footer;
