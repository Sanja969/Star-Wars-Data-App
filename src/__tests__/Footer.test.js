import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Footer from '../components/Footer';

describe('Test Footer render', () => {
  it('Test Footer snapshot', () => {
    const tree = renderer
      .create(
        <Router>
          <Footer />
        </Router>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
