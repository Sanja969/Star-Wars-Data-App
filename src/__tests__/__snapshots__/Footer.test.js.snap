// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer render Test Footer snapshot 1`] = `
<footer
  className="text-white d-flex align-items-center justify-content-around"
>
  <button
    className="logo-footer"
    onClick={[Function]}
    type="button"
  >
    Star Wars
  </button>
  <a
    className="text-decoration-none text-white"
    href="https://sanja42.pythonanywhere.com/"
  >
    COPYRIGHT Â© SANJA969
  </a>
</footer>
`;
